backend/routes/router.ts

```diff
@@ -82,6 +83,7 @@ router.use('/course', courseRouter)
 courseRouter.get('/create', createCourse)
 courseRouter.get('/read', readCourse)
 courseRouter.get('/all', readAllCourses)
+courseRouter.get('/search/:text', getSearchedCourses)
```

Comment:
```suggestion
courseRouter.get('/search/', getSearchedCourses)
```

にしたいですね！
今後複数のパラメータをフロントから渡してくる場合汎用性が無さそうなので！

Reply:
@issiyda 
getの場合、bodyで値が渡せませんが
どのようにして複数のパラメータを渡すのでしょうか？